From 9b54be32aba8df37f3255f5e31a99968c657711e Mon Sep 17 00:00:00 2001
From: Haohui Mai <wheat9@apache.org>
Date: Wed, 29 Oct 2014 17:54:26 -0700
Subject: [PATCH 746/791] HADOOP-11247. Fix a couple javac warnings in NFS.
 Contributed by Brandon Li.

Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt
---
 .../apache/hadoop/security/IdMappingConstant.java  |    2 +-
 .../org/apache/hadoop/nfs/nfs3/Nfs3Constant.java   |    8 --------
 .../hadoop/hdfs/nfs/conf/NfsConfiguration.java     |    4 ++--
 .../hadoop/hdfs/nfs/nfs3/RpcProgramNfs3.java       |    2 +-
 4 files changed, 4 insertions(+), 12 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/IdMappingConstant.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/IdMappingConstant.java
index f43556f..29664fb 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/IdMappingConstant.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/IdMappingConstant.java
@@ -32,5 +32,5 @@
   
   // Used for finding the configured static mapping file.
   public static final String STATIC_ID_MAPPING_FILE_KEY = "static.id.mapping.file";
-  public static final String STATIC_ID_MAPPING_FILE_DEFAULT = "/etc/usergroupid.map";
+  public static final String STATIC_ID_MAPPING_FILE_DEFAULT = "/etc/nfs.map";
 }
diff --git a/hadoop-common-project/hadoop-nfs/src/main/java/org/apache/hadoop/nfs/nfs3/Nfs3Constant.java b/hadoop-common-project/hadoop-nfs/src/main/java/org/apache/hadoop/nfs/nfs3/Nfs3Constant.java
index 8b269af..6ab6f6f 100644
--- a/hadoop-common-project/hadoop-nfs/src/main/java/org/apache/hadoop/nfs/nfs3/Nfs3Constant.java
+++ b/hadoop-common-project/hadoop-nfs/src/main/java/org/apache/hadoop/nfs/nfs3/Nfs3Constant.java
@@ -193,12 +193,4 @@ public static WriteStableHow fromValue(int id) {
   /** Expiration time for nfs exports cache entry */
   public static final String NFS_EXPORTS_CACHE_EXPIRYTIME_MILLIS_KEY = "nfs.exports.cache.expirytime.millis";
   public static final long NFS_EXPORTS_CACHE_EXPIRYTIME_MILLIS_DEFAULT = 15 * 60 * 1000; // 15 min
-
-  @Deprecated
-  public final static String NFS_USERGROUP_UPDATE_MILLIS_KEY = "nfs.usergroup.update.millis";
-  
-  // Used for finding the configured static mapping file.
-  @Deprecated
-  public static final String NFS_STATIC_MAPPING_FILE_KEY = "nfs.static.mapping.file";
-  public static final String NFS_STATIC_MAPPING_FILE_DEFAULT = "/etc/nfs.map";
 }
diff --git a/hadoop-hdfs-project/hadoop-hdfs-nfs/src/main/java/org/apache/hadoop/hdfs/nfs/conf/NfsConfiguration.java b/hadoop-hdfs-project/hadoop-hdfs-nfs/src/main/java/org/apache/hadoop/hdfs/nfs/conf/NfsConfiguration.java
index 5e73afb..23f9148 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-nfs/src/main/java/org/apache/hadoop/hdfs/nfs/conf/NfsConfiguration.java
+++ b/hadoop-hdfs-project/hadoop-hdfs-nfs/src/main/java/org/apache/hadoop/hdfs/nfs/conf/NfsConfiguration.java
@@ -43,9 +43,9 @@ private static void addDeprecatedKeys() {
             Nfs3Constant.NFS_EXPORTS_CACHE_EXPIRYTIME_MILLIS_KEY),
         new DeprecationDelta("hadoop.nfs.userupdate.milly",
             IdMappingConstant.USERGROUPID_UPDATE_MILLIS_KEY),
-        new DeprecationDelta(Nfs3Constant.NFS_USERGROUP_UPDATE_MILLIS_KEY,
+        new DeprecationDelta("nfs.usergroup.update.millis",
             IdMappingConstant.USERGROUPID_UPDATE_MILLIS_KEY),
-        new DeprecationDelta(Nfs3Constant.NFS_STATIC_MAPPING_FILE_KEY,
+        new DeprecationDelta("nfs.static.mapping.file",
             IdMappingConstant.STATIC_ID_MAPPING_FILE_KEY),
         new DeprecationDelta("dfs.nfs3.enableDump",
             NfsConfigKeys.DFS_NFS_FILE_DUMP_KEY),
diff --git a/hadoop-hdfs-project/hadoop-hdfs-nfs/src/main/java/org/apache/hadoop/hdfs/nfs/nfs3/RpcProgramNfs3.java b/hadoop-hdfs-project/hadoop-hdfs-nfs/src/main/java/org/apache/hadoop/hdfs/nfs/nfs3/RpcProgramNfs3.java
index fb21565..91d066e 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-nfs/src/main/java/org/apache/hadoop/hdfs/nfs/nfs3/RpcProgramNfs3.java
+++ b/hadoop-hdfs-project/hadoop-hdfs-nfs/src/main/java/org/apache/hadoop/hdfs/nfs/nfs3/RpcProgramNfs3.java
@@ -174,7 +174,7 @@ public RpcProgramNfs3(NfsConfiguration config, DatagramSocket registrationSocket
     this.config = config;
     config.set(FsPermission.UMASK_LABEL, "000");
     iug = new ShellBasedIdMapping(config,
-        Nfs3Constant.NFS_STATIC_MAPPING_FILE_DEFAULT);
+        IdMappingConstant.STATIC_ID_MAPPING_FILE_DEFAULT);
 
     aixCompatMode = config.getBoolean(
         NfsConfigKeys.AIX_COMPAT_MODE_KEY,
-- 
1.7.9.5

